GSLabMisc/tests/testthat/input/txt/diverse.txt

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Do we need the /input/txt sub or can we flatten to /input?

Reply:
@ew487 

> @jmshapir I think we can flatten it which I did in [13afd84](https://github.com/gslab-econ/gslab_r/pull/26/commits/13afd841a4f5931443458ef5ee2d44ec4f6cde61). 

Great, thanks.

> I was wondering, can we also remove `input/combined_data` and `input/lib`? They don't seem to have any files in them.

I don't think git recognizes empty subs, and I'm not seeing them in [/input](https://github.com/gslab-econ/gslab_r/tree/13afd841a4f5931443458ef5ee2d44ec4f6cde61/GSLabMisc/tests/testthat/input). (See also below.) Can you clarify?

![image](https://user-images.githubusercontent.com/7840353/215290165-79c59744-5d28-48d0-bc35-f104ba643813.png)

Reply:
@jmshapir My bad, it must just exist on my local computer then. I will get rid of it!